conversationgenome/analytics/WandbLib.py

```diff
@@ -26,53 +26,42 @@
 class WandbLib:
     verbose = False
 
-    def init_wandb(self, data=None):
+    def init_wandb(self, config=None, data=None):
+        if config:
+            #initialize data:
+            wallet = bt.wallet(config=config)
+            subtensor = bt.subtensor(config=config)
+            metagraph = subtensor.metagraph(config.netuid)
+            my_hotkey=wallet.hotkey.ss58_address
+            my_uid = metagraph.hotkeys.index(my_hotkey)
+        else: 
+            #initialize test data:
+            my_hotkey=12345
+            my_uid = -1
+
         if c.get("env", "WANDB_DISABLE"):
             return
         api = wandb.Api()
         wandb_api_key = c.get("env", "WANDB_API_KEY")
         if not wandb_api_key:
             raise ValueError("Please log in to wandb using `wandb login` or set the WANDB_API_KEY environment variable.")
-        run = 5
+
         bt.logging.info("INIT WANDB", wandb_api_key)
 
-        my_uid = 2
-        PROJECT_NAME = 'cgp_test_run'
+        PROJECT_NAME = 'conversationgenome'
```

Comment:
This should probably be a config with an IF statement for test runs

Reply:
For now, i added a netuid parameter to the logs for easy filtering